product_attribute_model_link/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Module name</h2>
+            <p>This module was written to extend the functionality of ... to support ... and allow you to ...</p>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Installation</h2>
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">To install this module, you need to:
+                <ul>
+                    <li>...</li>
+                </ul>
+            </p>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_demo oe_picture oe_screenshot">
+                <a href="https://www.odoo.com/saas_master/demo?lang=en_US&module=crm">
+                    <img src="crm_sc_01.png">
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Configuration</h2>
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">To configure this module, you need to:
+                <ul>
+                    <li>...</li>
```

Comment:
... ?

Reply:
Thanks for the review. Any thoughts on why it's not filled in the pre-commit? Tried a few times, but it's not working. Appreciate your help!